### ISSUE TITLE
Low accuracy  in small dataset

### ISSUE THREAD

**Issue by liqier** [xiaoiker/GCN-NAS#13]
Hello, thank you for your work. I applied your network to a smaller data set (400 samples, 12 categories), but the recognition results were poor. The following is my training process. How can I improve this situation?
![image](https://user-images.githubusercontent.com/55725190/111758228-3268b080-88d7-11eb-8924-d2f14099530a.png)
![image](https://user-images.githubusercontent.com/55725190/111757969-e0c02600-88d6-11eb-8064-0989ff6b7aa9.png)
![image](https://user-images.githubusercontent.com/55725190/111758029-f3d2f600-88d6-11eb-83f7-c15fc3dc882f.png)

**Comment by xiaoiker** [comment]
Hey, thanks for your interests to our work. This could happen since there are very limited smaples. Another thing is I am not sure whether you configure all the settings correctly.

My suggestions are: 1. make a compact model by removing some layers. 2. if it still not work, I suggest you use the original ST-GCN to check whether this issue is caused by our model or GCN is not work for your case. 3. If GCNs really not work but you still want to apply GCN to your task, find some data augmentation methods to add some training samples.

Best,

Wei

**Comment by liqier** [comment]
Thank you for your reply. Do you have any suggestions for the design of config for small-scale dataset?
Looking forward to your reply！

**Comment by xiaoiker** [comment]
I didnot try this, but you can test a GCN with 3 layers, and the feature dimensions can be set like 3(input)-->64-->64-->128-->FC.  You should test many similar configs before you find an optimal one. 

Best,
Wei

**Comment by liqier** [comment]
Thank you for your suggestion. I have another question: can GCN use a trained model as a preprocessing model for a new dataset like the convolutional neural network? That is, can I use the pre-trained model (for example, weights.pt trained by kinetics) as the pre-processing model for a new dataset (different from the kinetics joint labeling method and different classification categories)?

Best,
Liqi

**Comment by xiaoiker** [comment]
When you say preprocessing model you mean pretrain model, right?  Sure, it can do the transfer learning as CNN.
